src/EnergyPlus/DXCoils.cc

```diff
@@ -526,14 +528,14 @@ void SimDXCoilMultiMode(EnergyPlusData &state,
             if ((state.dataDXCoils->DXCoil(DXCoilNum).NumCapacityStages == 1) || (PartLoadRatio <= 0.0)) {
                 // Run stage 1 at its part load
                 PerfMode = DehumidMode * 2 + 1;
-                CalcDoe2DXCoil(state, DXCoilNum, On, FirstHVACIteration, PartLoadRatio, FanOpMode, PerfMode);
+                CalcDoe2DXCoil(state, DXCoilNum, CompressorOperation::On, FirstHVACIteration, PartLoadRatio, FanOpMode, PerfMode);
                 S1PLR = PartLoadRatio;
                 S2PLR = 0.0;
             } else {
                 // If a two-stage coil
                 // Run stage 1 at full load
                 PerfMode = DehumidMode * 2 + 1;
-                CalcDoe2DXCoil(state, DXCoilNum, On, FirstHVACIteration, 1.0, FanOpMode, PerfMode);
+                CalcDoe2DXCoil(state, DXCoilNum, CompressorOperation::On, FirstHVACIteration, 1.0, FanOpMode, PerfMode);
```

Comment:
Not part of this PR, but I would also like to see some kind of pattern where non-symbolic arguments are documented inline, e.g.,: 

```cpp
CalcDoe2DXCoil(state, DXCoilNum, CompressorOperation::On, FirstHVACIteration, /* PartLoadRatio */1.0, FanOpMode, PerfMode);
```

Or 

```cpp
CalcDoe2DXCoil(state, DXCoilNum, CompressorOperation::On, FirstHVACIteration, 
                             1.0, // PartLoadRatio 
                             FanOpMode, PerfMode);
```

Reply:
I like the first one, and never even thought of doing that.

Reply:
I like the first one also, but I think we are trying to avoid using C-style comments. @Myoldmopar, can you advise?

Reply:
Correct, we agreed to make C style comments illegal because they make parsing the code during refactors more difficult.  We have a CI script that will fail if you add C style arguments, so the C++ style will be the way to go.